app/views/layouts/_head.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= tag.head :data => application_data do %>
   <%= render :partial => "layouts/meta" %>
   <%= javascript_include_tag "turbo", :type => "module" %>
+  <%= javascript_include_tag "make-plural/cardinals" %>
```

Comment:
What's the reason for loading this as a separate file rather than requiring it in `application.js`?

Reply:
Name collisions. They have `const a`, `const b` etc defined in the global scope. When I added `make-plural/cardinals` as `//= require ...` in `application.js` it was picking wrong pluralization rules.

Reply:
Obviously that global namespace pollution is poor but I don't see why it would break anything, and having tried it myself I can't reproduce any problem once the assets are properly rebuilt.

Reply:
When I started doing this, we were still using vars. It could be var redeclarations, which are no longer happening, because now we're using let/const.

Reply:
Changed to `//= require`